kratos/python_scripts/testing/utilities.py

```diff
@@ -117,7 +118,7 @@ def RunMPITestSuit(self, application, path, mpi_command, mpi_flags, num_processe
                     full_command
                 ], shell=True,
                    stdout=subprocess.PIPE,
-                   cwd=os.path.dirname(os.path.abspath(test_script)))
+                   cwd=directory)
```

Comment:
```suggestion
                   cwd=os.path.dirname(os.path.abspath(str(test_script))))
```

why not like this?

Reply:
not sure if that will work, i can give it a try

Reply:
it works, commiting.